src/maidsafe/routing/connection_manager.h

```diff
@@ -163,7 +163,7 @@ class ConnectionManager {
   PublicPmid our_fob_;
   Address our_id_;
 
-  std::map<unsigned short, std::unique_ptr<crux::acceptor>> acceptors_;
+  std::map<unsigned short, std::unique_ptr<crux::acceptor>> acceptors_;  // NOLINT
```

Comment:
I added // NOLINT as elsewhere unsigned short was used in similar context, but equally style check suppressed.